test/e2e/pkg/grammar/clean-start/abci_grammar_clean_start.md

```diff
@@ -0,0 +1,33 @@
+```
+package "github.com/cometbft/cometbft/test/e2e/pkg/grammar/clean-start"
+
+
+Start : CleanStart ;
```

Comment:
Open-ended question: since you had to split the original grammar into two, isn't that a sign that the original grammar (in the spec) should be similarly split? (something to reflect on)